datadog-slos-slo-handler/src/datadog_slos_slo/handlers.py

```diff
@@ -0,0 +1,234 @@
+import logging
+from typing import Any, List, MutableMapping, Optional
+
+from cloudformation_cli_python_lib import (
+    Action,
+    HandlerErrorCode,
+    OperationStatus,
+    ProgressEvent,
+    Resource,
+    SessionProxy,
+)
+from datadog_api_client.v1 import ApiException
+from datadog_api_client.v1.api.service_level_objectives_api import ServiceLevelObjectivesApi
+from datadog_api_client.v1.model.service_level_objective import ServiceLevelObjective as ApiSLO
+from datadog_api_client.v1.model.service_level_objective_query import ServiceLevelObjectiveQuery as \
+    ApiSLOQuery
+from datadog_api_client.v1.model.slo_threshold import SLOThreshold as ApiSLOThreshold
+from datadog_api_client.v1.model.slo_timeframe import SLOTimeframe as ApiSLOTimeframe
+from datadog_api_client.v1.model.slo_type import SLOType as ApiSLOType
+from datadog_api_client.v1.model.slo_list_response import SLOListResponse as ApiSLOList
+from datadog_api_client.v1.model.service_level_objective_request import \
+    ServiceLevelObjectiveRequest as ApiSLORequest
+from datadog_cloudformation_common.api_clients import v1_client
+
+from .models import Creator, Threshold, Query, ResourceHandlerRequest, ResourceModel
+from .version import __version__
+
+# Use this logger to forward log messages to CloudWatch Logs.
+LOG = logging.getLogger(__name__)
+TYPE_NAME = "Datadog::SLOs::SLO"
+TELEMETRY_TYPE_NAME = "slos-slo"
+
+resource = Resource(TYPE_NAME, ResourceModel)
+test_entrypoint = resource.test_entrypoint
+
+
+@resource.handler(Action.READ)
+def read_handler(
+        session: Optional[SessionProxy],
+        request: ResourceHandlerRequest,
+        callback_context: MutableMapping[str, Any],
+) -> ProgressEvent:
+    LOG.info("Starting %s Read Handler", TYPE_NAME)
+    model = request.desiredResourceState
+    with v1_client(
+            model.DatadogCredentials.ApiKey,
```

Comment:
This will also need to be changed to use `request.typeConfiguration`

Reply:
This has been updated